### DISCUSSION TITLE
It appears that an Eloquent Model's column was updated and is being used at the same time for mass filtering

### DISCUSSION THREAD

**Discussion by mondersky** [iksaku/laravel-mass-update#4] [Q&A] [answered]
Hello, I am trying to perform a simple mass update
       Calendar::massUpdate([["name" => "updated name", "google_id" => "abc"]],uniqueBy: [''google_id']);
       
I am getting this error message:
**It appears that an Eloquent Model's column was updated and is being used at the same time for mass filtering. This may cause filtering issues and may not even update the value properly. Affected columns: - abc**

I don't really understand what the message means by "column was updated and is being used at the same time", apart the fact that the column "google_id" is present in both the first and the second argument at the same time. Which makes no sens to me unless there's something I am missing.
anyone can tell me what I am doing wrong ? please
Laravel 8

**Comment by iksaku** [comment, marked as answer]
Hello @mondersky.

It looks like you're trying to update an Eloquent Model to assign a new value to a column while also using that same value as a partitioning key. In order to prevent execution errors, we don't allow such strategy.

Would you mind sharing the _actual_ piece of code you are trying to execute? This error will only appear when passing an array of models instead of a raw array of values.

Regards

**Reply by mondersky** [reply]
In fact it was because I was passing an array of models rather than an array, thank you very much for your help !
